### PR TITLE
Feat/duplicate

### PR DESCRIPTION
**Pull request by brithe00**

### CODE REVIEW

File: src/components/SearchFighter.jsx [hunk 1]
```diff
@@ -7,8 +7,10 @@ import './SearchFighter.css';
 const SearchFighter = ({
   setKeywords,
   onSearch,
+  label
 }) => (
   <div className="SeachFighter">
+    <>{label}</>
```
Comment (rocambille) [inline src/components/SearchFighter.jsx]: maybe use a \<label\> for your label ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by brithe00** [comment]
Like that ?

**Comment by rocambille** [comment]
this should be improved: e.g. the label should be linked to a control (input, select, textarea) using htmlFor

but that's it :)

**Comment by brithe00** [comment]
Ok, I see, it can be improved with this for example :

`<label htmlFor="fighterInput">Fighter :</label>`

**Comment by rocambille** [comment]
yes, but you will need something identified as fighterInput. something like that:

    <label htmlFor="fighterInput">Fighter :</label>
    <input id="fighterInput" />

**Pull request merged**